pallas-codec/src/utils.rs

```diff
@@ -665,7 +665,7 @@ impl<C, T> minicbor::Encode<C> for KeepRaw<'_, T> {
 /// ```
 #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
 pub struct AnyCbor {
-    inner: Vec<u8>,
+    pub inner: Vec<u8>,
```

Comment:
why do we need this?
If you want read-only access you can use `raw_bytes()`
if you want to get the vec you can use `unwrap()`

Reply:
the inner is created only when it has a ready to use instance,

in my case I already have the bytes so was doing something like

```
let datum = AnyCbor { inner: bytes };
```